tests/clone_test.go

```diff
@@ -839,22 +694,22 @@ var _ = Describe("VirtualMachineClone Tests", Serial, func() {
 	})
 })
 
-func startCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (*virtv1.VirtualMachine, error) {
-	patch, err := patch.New(patch.WithAdd("/spec/runStrategy", virtv1.RunStrategyAlways)).GeneratePayload()
+func startCloneVM(vmName, vmNamespace string) (*virtv1.VirtualMachine, error) {
+	patchBytes, err := patch.New(patch.WithAdd("/spec/runStrategy", virtv1.RunStrategyAlways)).GeneratePayload()
 	if err != nil {
 		return nil, err
 	}
 
-	return virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
+	return kubevirt.Client().VirtualMachine(vmNamespace).Patch(context.Background(), vmName, types.JSONPatchType, patchBytes, v1.PatchOptions{})
 }
```

Comment:
Thinking a little more about this. Why don't we simply replace this helper with a call to `kubevirt.Client().VirtualMachine(vmNamespace).Start()` and drop the helper?

Reply:
I agree. will remove.